src/graphic/Fast3D/gfx_gx2.cpp

```diff
@@ -135,6 +135,10 @@ static const char* gfx_gx2_get_name() {
     return "GX2";
 }
 
+static int gfx_gx2_get_max_texture_size() {
+    return 8192;
```

Comment:
This can probably be hardcoded since it's a console, but I'd assume gx2 also has a way to query this.

Reply:
Not aware of any ways to query this. Could be turned into a define in the Wii U toolchain.
I think it's fine to hardcode it for now.

Reply:
If there's no define in the WII U toolchain this is fine for now.